PDF Watermark Remover/readme.md

```diff
@@ -0,0 +1,33 @@
+# PDF-Watermark-Remover
+
+The script PDF-Watermark-Remover will remove the watermark from the PDF files. The watermark is selected based on ranges of RGB color and based on selected pixels it will remove the watermark from the pages.
+
+## Setup instructions
+1. Download the python script file ("PDF-Watermark-Remover.py")
+2. Open the cmd/powershell/shell and type below line & hit return
+    - For windows
+        ```
+        python <path of "PDF-Watermark-Remover.py">
+        ```
+    - For mac/linux
+        ```
+        python3 <path of "PDF-Watermark-Remover.py">
+        ```
+
+## Output
+
+| Original Page | Output Page |
+| --- | --- |
+| <image src= "Sample\sample001.png" width = 2000px> | <image src= "Results\img1.jpg" width = 2000px> |
+| <image src= "Sample\sample002.png" width = 2000px> | <image src= "Results\img2.jpg" width = 2000px> |
+| <image src= "Sample\sample003.png" width = 2000px> | <image src= "Results\img3.jpg" width = 2000px> |
+| <image src= "Sample\sample004.png" width = 2000px> | <image src= "Results\img4.jpg" width = 2000px> |
+| <image src= "Sample\sample005.png" width = 2000px> | <image src= "Results\img5.jpg" width = 2000px> |
+| <image src= "Sample\sample006.png" width = 2000px> | <image src= "Results\img6.jpg" width = 2000px> |
+
+## Author(s)
+Zankrut Goyani <br>
```

Comment:
```suggestion
[Zankrut Goyani](https://github.com/zankrut20)
```

Reply:
No, I'm not the GSSOC program participant. Sorry to say but I heard it the first time and also surprised with that tag.

Changes are done as suggested. I'm not uploading images on Imgur because a readme sample is sufficient.